CM Breaking changes in 6.7 (#11434)
elastic/beats: libbeat/docs/breaking.asciidoc

include::./field-name-changes.asciidoc[]

[[breaking-changes-6.7]]
=== Breaking changes in 6.7

> The Central management data structure used by Kibana to store the information has changed. As a result of the

Reviewer: ```suggestion
The data structure used to store Beats central management data in Kibana has changed. As a result of the
```

> template changes, enrolled beats and tags created prior to 6.7 will not work with the new

Reviewer: ```suggestion
template changes, enrolled Beats and configuration tags created prior to 6.7 will not work until you migrate them to the new format.
```

> the new format, we have created a one time [tool]() to help you migrate your data from

Author: @kvch I will use your link here.

> UI until their are moved to the new format, we have created a one time [tool](https://github.com/elastic/migrate-management-beats/releases) to help you migrate your data from

Reviewer: ```suggestion
```suggestion
To migrate to the new format, use the https://github.com/elastic/migrate-management-beats/releases[migration tool] we've created.
```
```

> UI until their are moved to the new format, we have created a one time [tool](https://github.com/elastic/migrate-management-beats/releases) to help you migrate your data from

Reviewer: ```suggestion
To migrate to the new format, use the https://github.com/elastic/migrate-management-beats/releases[migration tool] we've created.
```

> 6.6 to 6.7, refer to the [documentation](https://github.com/elastic/migrate-management-beats/blob/master/README.md) for instruction. {see-relnotes}

Reviewer: ```suggestion
For more information, see the https://github.com/elastic/migrate-management-beats/blob/master/README.md[documentation] about the tool. {see-relnotes}
```
